migrations/0030-compute-node-sets-b1869c.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2016 Eugene Frolov <eugene@frolov.net.ru>
+# Copyright 2025 Genesis Corporation
+#
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from restalchemy.storage.sql import migrations
+
+
+class MigrationStep(migrations.AbstarctMigrationStep):
+
+    def __init__(self):
+        self._depends = ["0029-sdk-0-7-0-migration-c2c52a.py"]
+
+    @property
+    def migration_id(self):
+        return "b1869cc2-82c0-42e8-a6a9-df07105c3df5"
+
+    @property
+    def is_manual(self):
+        return False
+
+    def upgrade(self, session):
+        sql_expressions = [
+            # TABLES
+            """
+            CREATE TABLE IF NOT EXISTS compute_sets (
+                "uuid" UUID NOT NULL PRIMARY KEY,
+                "project_id" UUID NOT NULL,
+                "name" varchar(255) NOT NULL,
+                "description" varchar(255) NOT NULL,
+                "cores" integer NOT NULL,
+                "ram" integer NOT NULL,
+                "root_disk_size" integer NOT NULL,
+                "replicas" integer NOT NULL,
+                "image" varchar(255) NOT NULL,
+                "node_type" VARCHAR(2) NOT NULL CHECK (node_type IN ('VM', 'HW')),
+                "set_type" VARCHAR(32) NOT NULL CHECK (set_type IN ('SET')),
+                "status" VARCHAR(32) NOT NULL CHECK (status IN ('NEW', 'SCHEDULED', 'IN_PROGRESS', 'STARTED', 'ACTIVE', 'ERROR')),
+                "nodes" varchar(1024) NOT NULL,
```

Comment:
I'm thinking how we will differ nodes, what do you think to add `node_set` relationship to nodes themselves? Or you think it's not explicit relationship and just optional?